src/screens/Feed/components/EmbeddedContent/component.js

```diff
@@ -204,26 +222,31 @@ class EmbeddedContent extends PureComponent {
       if (embed.url.includes('users')) return this.renderUser(embed);
     }
 
-    return null;
+    return this.renderLink(embed);
```

Comment:
I can't comment above for some reason but this line:

```
<WebComponent
           style={style}
           source={{ uri: `${video.url}?rel=0&autoplay=0&showinfo=1&controls=1&modestbranding=1` }}
/>
```

This will break when the `video.url` has timestamp information in it. You'll need to check if `?` already exists.

Example: `https://www.youtube.com/embed/c9FIvUcjhFg?start=80`